script.js

```diff
@@ -0,0 +1,71 @@
+let completedCells = 0;
+const ticTacToeBoard = [];
+
+// this is the main function of the game, updating the status of the game
+// gets the current player, updates the board and marks the cell that was completed
+function updateStatus(currentPlayer) {
+    if (completedCells % 2 == 0 && currentPlayer.id !== "completed") {
+        updateBoard(currentPlayer, 'X');
+        currentPlayer.innerText = "X";
+        ++completedCells;
+    } else if (currentPlayer.id !== "completed" && completedCells % 2 == 1) {
+        updateBoard(currentPlayer, '0');
+        currentPlayer.innerText = "0";
+        ++completedCells;
+    }
+    changeId(currentPlayer);
+    winnerUpdate();
+}
+
+// the game update function "updateStatus" will be called for each cell in the table
+window.onload = function completeBoard() {
```

Comment:
Let's put the `<script>` tag at the bottom of the `<body>` tag so we don't need to use `window.onload`.

Also, let's put this code at the beginning of the script.